app/services/unsent_documents_service.rb

```diff
@@ -0,0 +1,34 @@
+
+
+class UnsentDocumentsService
+  include ZendeskServiceHelper
+  include Rails.application.routes.url_helpers
+
+  def instance
+    @instance ||= EitcZendeskInstance
+  end
+
+  def detect_unsent_docs_and_notify
+    tickets_updated = 0
+    Intake.where.not(intake_ticket_id: nil).find_each(batch_size: 100) do |intake|
+      unsent_docs = intake.documents.where(zendesk_ticket_id: nil).where("created_at < ?", 15.minutes.ago)
+      if unsent_docs.present?
+        comment_body = <<~BODY
+          New client documents are available to view: #{zendesk_ticket_url(id: intake.intake_ticket_id)}
+          Files uploaded:
+          #{unsent_docs.map {|d| "* #{d.upload.filename} (#{d.document_type})"}.join("\n")}
+        BODY
+
+        append_comment_to_ticket(
```

Comment:
I wonder if we should add the zendesk_ticket_url to the custom field as well? Might be a question for @bengolder or   Kelly.

Reply:
Yes we should, that is a good catch. but non-blocking